app/common/formatters/TimestampFormatter.scala

```diff
@@ -0,0 +1,15 @@
+package formatters
+
+import java.sql.Timestamp
+import play.api.libs.json.{Format, JsResult, JsValue, Json}
+
+trait TimestampFormatter {
```

Comment:
was needed due to no implicit format available for java.sql.Timestamp conversion compile time warning/error while applying Json.toJson implicit formatting.

Reference taken from : [PublicStackODiscussion](https://stackoverflow.com/questions/28812503/scala-playframework-implicit-reader-writer-for-timestamp)